.github/workflows/test.yml

```diff
@@ -8,25 +8,34 @@ on:
     - '*'
     paths-ignore:
     - README.md
+  schedule:
+    - cron: '0 7 * * SUN'
 jobs:
   test:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu, macos, windows ]
+        os: [ ubuntu-16.04, ubuntu-18.04, ubuntu-20.04, macos-10.15, windows-2016, windows-2019 ]
```

Comment:
Could you remove `windows-2016` here since it's not supported (see #61 and the README)?
And then many excludes are no longer needed.

Reply:
See https://github.com/actions/virtual-environments#available-environments, there it is still listed as available environment, not sure which docu is now correct, but windows-2016 does seem to work